### DISCUSSION TITLE
有办法对 edge 的可移动范围进行限制吗？

### DISCUSSION THREAD

**Discussion by ghost** [antvis/X6#1319] [Q&A] [answered]
有办法对 edge 的可移动范围进行限制吗？文档里提到的好像仅针对 node 有效
 
https://x6.antv.vision/zh/docs/tutorial/basic/interacting/#restrict

比如说，我设置 restrict 为 { x: 0, y: 0, width: Infinity, height: Infinity } 可以使节点在 x >= 0 && y >= 0 的范围内移动
但是 edge 似乎不受这个规则限制

**Comment by NewByVector** [comment, marked as answer]
暂时不支持对 edge 移动进行限制。
